Gemfile

```diff
@@ -1,30 +1,32 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
 
-ruby "3.1.4"
+ruby '3.1.4'
+
+gem 'rubocop', '>= 1.0', '< 2.0'
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
-gem "rails", "~> 7.1.1"
+gem 'rails', '~> 7.1.1'
 
 # The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
-gem "sprockets-rails"
+gem 'sprockets-rails'
 
 # Use sqlite3 as the database for Active Record
-gem "sqlite3", "~> 1.4"
+gem 'sqlite3', '~> 1.4'
 
```

Comment:
- [ ] Since we'll be using PostgreSQL instead of SQLite I'd suggest you replace the above gem file with the one shown below 

```gem
# Use postgresql as the database for Active Record
gem 'pg', '~> 1.1'
```